### PR TITLE
added popularity and dead relation too main file

### PR DESCRIPTION
**Pull request by Hack3l**

### CODE REVIEW

File: api-handler/dataAccess.js [hunk 1]
```diff
@@ -5,12 +5,16 @@ const cul = require("./getAllCultures");
 const hou = require("./getAllHouses");
 const reg = require("./getAllRegions");
 const tit = require("./getAllTitles");
+const pop = require("./getPopularity");
+const rel = require("./support/deadRelations");
 
 const url_cha = 'https://got-api.bruck.me/api/characters/';
```
Comment (sacdallago) [inline api-handler/dataAccess.js:11]: I hope you make this configurable! The API is not gonna be serviced on bruck's server in the final round!

Comment (Hack3l) [inline api-handler/dataAccess.js:11]: populartity is currently gotten from the file u provided and the relation to dead characters is getting scrapped from the wiki so there are no api calls being made at the moment

Comment (sacdallago) [inline api-handler/dataAccess.js:11]: Still :D I don't like seeing bruck's server hardcoded anywhere, no matter if or not used :D Actually, if it's not used, why is it even there?

Comment (Hack3l) [inline api-handler/dataAccess.js:11]: Ah u meant the other things thought you meant for popularity and dead relations

**Pull request merged**